src/google_map.js

```diff
@@ -10,6 +10,7 @@ import MarkerDispatcher from './marker_dispatcher';
 import GoogleMapMap from './google_map_map';
 import GoogleMapMarkers from './google_map_markers';
 import GoogleMapMarkersPrerender from './google_map_markers_prerender';
+import * as GoogleHeatmap from './google_heatmap';
```

Comment:
Lets specify what we want to import, instead of importing everything, just to be sure.